test/concurrent/WaitLockTest.ooc

```diff
@@ -15,7 +15,7 @@ WaitLockTest: class extends Fixture {
 		super("WaitLock")
 		this add("_testWithMutexOwnership", This _testWithMutexOwnership)
 		this add("_testWithoutMutexOwnership", This _testWithoutMutexOwnership)
-		this add("_testWakeWithFailingCondition", This _testWakeWithFailingCondition)
+		version (!windows) { this add("_testWakeWithFailingCondition", This _testWakeWithFailingCondition) }
```

Comment:
Yeah I forgot to tell you, can you add `android` here as well while you're at it ? thanks :)